tritium-metrics-jvm/src/test/java/com/palantir/tritium/metrics/jvm/JvmMetricsTest.java

```diff
@@ -82,7 +82,8 @@
             "jvm.threads.terminated.count",
             "jvm.threads.blocked.count",
             "os.load.1",
-            "os.load.norm.1"
+            "os.load.norm.1",
+            "os.process.cpu.utilization"
```

Comment:
What if we called this `jvm.cpu.utilization`, since it applies only to this JVM?
The os prefix is a bit confusing as in the other 2 cases, it refers to OS-wide metrics.

Reply:
Possibly, but `process.cpu.utilization` might be better, we could compare this between java/rust/go services, not just those running in a jvm. Thoughts?